ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vxlan-interface.j2

```diff
@@ -14,6 +14,9 @@ interface Vxlan1
 {%     if vxlan_interface.Vxlan1.vxlan.source_interface is arista.avd.defined %}
    vxlan source-interface {{ vxlan_interface.Vxlan1.vxlan.source_interface }}
 {%     endif %}
+{%     if vxlan_interface.Vxlan1.vxlan.multicast_headend_replication is arista.avd.defined(true) %}
+   vxlan multicast headend-replication
+{%     endif %}
```

Comment:
After testing on a device, this command should be at the very end of the vxlan config section

Reply:
Thanks @ClausHolbechArista, fixing it.